apis/Google.Cloud.Firestore.Data/docs/index.md

```diff
@@ -8,4 +8,14 @@
 
 # Getting started
 
-Samples will be added when the API surface is more stable.
+Everything starts with [FirestoreDb](obj/api/Google.Cloud.Firestore.Data.FirestoreDb.yml).
+Create an instance using the `Create` or `CreateAsync` methods, passing in your Google Cloud project ID.
+The default credentials will be used to authenticate with the server.
```

Comment:
Is there a way to specify credentials?

Reply:
Not simply, no - in the same way that there isn't for the gRPC client itself (you have to provide a channel).
You'd need to create a suitable `FirestoreClient` and pass that in. (I'm not going into that aspect right now.)